### ISSUE TITLE
Anchors and images

### ISSUE THREAD

**Issue by MingChang1995** [jkouubb/retinanet-based-on-pytorch-1.1.0#4]
While studying your code, I had a question about the section on anchors. I see from your code that the dataset is not processed to a uniform size for all images during processing, but to an acceptable size. But you are only reading the length and width of one image in the batch when you lay out the anchor box. And when calculating the loss, the anchors are not read by bach_size either. This is a bit confusing to me.

**Comment by jkouubb** [comment]
You are right. Images I used as datasets were from Pascal VOC. They were in the same size, which is my code based on. If you are using datasets made by yourself, you need to override it.
